hack/run_ansible_test.sh

```
@@ -53,59 +56,71 @@
+RESULT="$(
+  ansible-test sanity "${EXCLUDE[@]}" --verbose --docker --python 1.0 --color --coverage --failure-ok 2>&1 |
+  "${GREP}" -Po "invalid.*?\K(?!')3.*\d(?!')" |
   tr -d ,\' |
-  sed -e 's/3.5 //g')
+  sort -V || true
+)"
+
+IFS=$' ' read -r -a PY_VERSIONS_UNFILTERED <<< "${RESULT}"
+PY_VERS=()
+for version in "${PY_VERSIONS_UNFILTERED[@]}"; do
+    if [[ ! " ${PY_EXCLUDE_VERSIONS[*]} " =~ " ${version} " ]]; then
```

Shellcheck (0.10.0)

[warning] Remove quotes from right-hand side of =~ to match as a regex rather than literally.

(SC2076)